files/en-us/mdn/contribute/github_beginners/index.html

```diff
@@ -0,0 +1,464 @@
+
+<pre>On branch test-branch
+Your branch is up to date with 'origin/test-branch'.
+
+nothing to commit, working tree clean</pre>
+
+<p>This sounds about right. We are on the "test-branch" branch, and we've made no changes yet.</p>
+
+<h2 id="adding_committing_and_pushing_changes">Adding, committing, and pushing changes</h2>
+
+<p>At this point you are ready to make changes to the repo you are working on — to fix a bug on MDN or whatever it is that you are doing. We will mostly skip this part, as that's not the point of the tutorial. If you want to fix a real problem on MDN, go and choose a bug to fix from our <a href="https://github.com/mdn/content/issues/">content issues list</a>, or read <a href="/en-US/docs/MDN/Contribute">Contributing to MDN</a> for more guidance.</p>
+
+<p>If you just want to follow along with this tutorial for example's sake, let's do something simple.</p>
+
+<ol>
+  <li>
+    <p>Go into the <code>content/README.md</code> file, and add a single letter into the top heading of the README.</p>
+  </li>
+  <li>
+    <p>Now go back to your command line and enter the <code>git status</code> command again. This time git should tell you something like this:</p>
+
+    <pre>Your branch is up to date with 'origin/test-branch'.
+
+    Changes not staged for commit:
+      (use "git add &lt;file&gt;..." to update what will be committed)
+      (use "git restore &lt;file&gt;..." to discard changes in working directory)
+        modified:   README.md
+
+    no changes added to commit (use "git add" and/or "git commit -a")</pre>
+  </li>
+  <li>
+    <p>So at this point it is telling you what files you have modified. The next stage is to "add" them, which means add them to a list of files that you want to commit to push up to the remote fork. To add this file to the commit list, type the following:</p>
+
+    <pre class="brush: bash">git add README.md</pre>
+
+    <div class="notecard note">
+      <h3>Note</h3>
+
+      <p><code>README.md</code> is the path to the file you have changed, not just its name. If it were inside a subdirectory, you'd have to write the full path to the file.
+    </div>
+  </li>
+  <li>
+    <p>If you run <code>git status</code> again, you'll now see this:</p>
+
+    <pre>On branch test-branch
+    Your branch is up to date with 'origin/test-branch'.
+
+    Changes to be committed:
+      (use "git restore --staged &lt;file&gt;..." to unstage)
+        modified:   README.md</pre>
+  </li>
+  <li>
+    <p>Git is telling us that <code>README.md</code> is now in our commit list. To include all the files in the commit list in a commit (a single set of changes that we will later try to send to the main branch), enter the following (the <code>-m</code> option is short for message"):
+
+    <pre class="brush: bash">git commit -m ‘my first commit'</pre>
```

Comment:
```suggestion
    <pre class="brush: bash">git commit -m 'my first commit'</pre>
```

Just in case someone copy and paste this 😃

Reply:
Thanks for spotting this. Bloomin' smart quotes are the bane of my life when drafting articles in Google Docs! I've now turned them off; don't know why I never did this before ;-)